lib/ui/fake_flutter/diagnosticable.dart

```diff
@@ -0,0 +1,2941 @@
+  manditoryFooter: '═' * 100,
+  /*FlutterError.wrapWidth,*/
+  // No need to add a blank line as the footer makes the boundary of this
+  // subtree unambiguous.
+  addBlankLineIfNoChildren: false,
+  isBlankLineBetweenPropertiesAndChildren: false,
+);
+
+/// Whitespace only configuration where children are consistently indented
+/// two spaces.
+///
+/// Use this style for displaying properties with structured values or for
+/// displaying children within a [transitionTextConfiguration] as using a style that
+/// draws line art would be visually distracting for those cases.
+///
+/// Example:
+/// ```
+/// <parent_node>
+///   <name>: <description>:
+///     <properties>
+///     <children>
+///   <name>: <description>:
+///     <properties>
+///     <children>
+/// ```
+///
+/// See also:
+///
+///  * [DiagnosticsTreeStyle.whitespace]
+final TextTreeConfiguration whitespaceTextConfiguration = TextTreeConfiguration(
+  prefixLineOne: '',
+  prefixLastChildLineOne: '',
+  prefixOtherLines: ' ',
+  prefixOtherLinesRootNode: '  ',
+  bodyIndent: '',
+  propertyPrefixIfChildren: '',
+  propertyPrefixNoChildren: '',
+  linkCharacter: ' ',
+  addBlankLineIfNoChildren: false,
+  // Add a colon after the description and before the properties to link the
+  // properties to the description line.
+  afterDescriptionIfBody: ':',
+  isBlankLineBetweenPropertiesAndChildren: false,
+);
+
+/// Render a node as a single line omitting children.
+///
+/// Example:
+/// `<name>: <description>(<property1>, <property2>, ..., <propertyN>)`
+///
+/// See also:
+///
+///  * [DiagnosticsTreeStyle.singleLine]
+final TextTreeConfiguration singleLineTextConfiguration = TextTreeConfiguration(
+  propertySeparator: ', ',
+  beforeProperties: '(',
+  afterProperties: ')',
+  prefixLineOne: '',
+  prefixOtherLines: '',
+  prefixLastChildLineOne: '',
+  lineBreak: '',
+  lineBreakProperties: false,
+  addBlankLineIfNoChildren: false,
+  showChildren: false,
+  propertyPrefixIfChildren: '',
+  propertyPrefixNoChildren: '',
+  linkCharacter: '',
+  prefixOtherLinesRootNode: '',
+);
+
+/// Render a node as a single line omitting children.
+///
+/// Example:
+/// ```
+/// <name>:
+///   <description>(<property1>, <property2>, ..., <propertyN>)
+/// ```
+///
+/// See also:
+///
+///  * [DiagnosticsTreeStyle.indentedSingleLine]
+final TextTreeConfiguration singleLineTextConfigurationIndented =
+    TextTreeConfiguration(
+  propertySeparator: ', ',
+  beforeProperties: '(',
+  afterProperties: ')',
+  prefixLineOne: '',
+  prefixOtherLines: '',
+  prefixLastChildLineOne: '',
+  lineBreak: '',
+  lineBreakProperties: false,
+  addBlankLineIfNoChildren: false,
+  showChildren: false,
+  propertyPrefixIfChildren: '',
+  propertyPrefixNoChildren: '',
+  linkCharacter: '',
+  prefixOtherLinesRootNode: '',
+  afterName:
+      '\n  ', // This is the difference between this text configuration and the typical single line text configuration. TODO(jacobr): verify this is robust.
```

Comment:
Is comment formatted differently on purpose? If so, maybe make that more clear in the text.

Reply:
I wouldn't worry about formatting for this file. it is copied straight from package flutter so we shouldn't tweak it unless strictly necessary.
I ran dartfmt here just so that we can continue to run dartfmt on our code base without diffs but dartfmt doesn't do a great job on this file.